### PR TITLE
Issue #448: Allow item list to have nested items.

### LINKED ISSUE #448: Bug Item list missing children (nested item)
The default theme has children, check [here](https://api.drupal.org/api/drupal/includes%21theme.inc/function/theme_item_list/7.x)

And Atomium 2.29 omit it.

This is causing an item list with nested items to not be displayed properly.

**Example facets in eiopa.**
**Right nested items**
screen_6item_list_hierarchy_in flat_mode_good_result

**Wrong nested items**
screen_3_item_list_hierarchy_in flat_mode

### PR DESCRIPTION
**Pull request by aritomelo**
Fixed #448

### CODE REVIEW

File: atomium/templates/item_list/item_list.component.inc [hunk 1]
```diff
@@ -128,6 +133,17 @@ function atomium_preprocess_item_list(array &$variables, $hook) {
       }
     }
 
+    if (isset($item['children'])) {
+      $children = $item['children'];
```
Comment (drupol) [inline atomium/templates/item_list/item_list.component.inc:137]: This line is not needed.

File: atomium/templates/item_list/item_list.component.inc [hunk 2]
```diff
@@ -128,6 +133,17 @@ function atomium_preprocess_item_list(array &$variables, $hook) {
...
+      unset($item['children']);
```
Comment (drupol) [inline atomium/templates/item_list/item_list.component.inc:138]: This line is not needed.

File: atomium/templates/item_list/item_list.component.inc [hunk 3]
```diff
@@ -128,6 +133,17 @@ function atomium_preprocess_item_list(array &$variables, $hook) {
...
+      $item['children'] = array(
+        '#theme' => 'item_list',
+        '#items' => $children,
```
Comment (drupol) [inline atomium/templates/item_list/item_list.component.inc:141]:
```suggestion
        '#items' => $item['children'],
```

**Review by drupol — CHANGES_REQUESTED** [review]

File: atomium/templates/item_list/item_list.component.inc (hunk 3 above)
Comment (aritomelo) [reply atomium/templates/item_list/item_list.component.inc:141]: The line is to swap the order been the main item and the children items, you can test the result with and without you will see!

**Pull request merged**